modules/docextract/lib/refextract_kbs.py

```diff
@@ -203,8 +201,6 @@ def institute_num_pattern_to_regex(pattern):
                             ('yy',   r'\d\d'),
                             ('s',    r'\s*'),
                             (r'/',   r'\/')]
-    # first, escape certain characters that could be sensitive to a regexp:
-    pattern = re_report_num_chars_to_escape.sub(r'\\\g<1>', pattern)
```

Comment:
what do you mean? this patch was tested against the testsuite of inspirehep/refextract so there should be no major regressions, but no further testing was done. What do you suggest?